hivemind/client/averaging/__init__.py

```diff
@@ -23,7 +23,7 @@
 StreamCallToLeader = grpc.aio.UnaryStreamCall[averaging_pb2.JoinRequest, averaging_pb2.MessageFromLeader]
 
 INITIAL_GROUP_NBITS = 3
-logger = get_logger(__file__)
+logger = get_logger(__name__)
```

Comment:
There is only one logger per module, and using __file__ instead of __name__ leads to incorrect module name resolution and duplicate handlers